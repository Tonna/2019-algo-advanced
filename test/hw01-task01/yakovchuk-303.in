25
1 1 2 1 1 4 4 4 8 6 2 6 7 12 9 2 8 16 17 1 19 3 22 12
3
16 5
1 10
22 7
