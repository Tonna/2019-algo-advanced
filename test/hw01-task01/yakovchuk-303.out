13
25
21
